g/go-jose/go-jose_2.5.1_ubi_8.5.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ex
+# -----------------------------------------------------------------------------
+#
+# Package	: go-jose
+# Version	: v2.5.1
```

Comment:
Please check if existing script go-jose_ubi_8.5.sh works for v2.5.1.

Reply:
just check the existing script against version v2.5.1 and the test cases are failing.

Reply:
LGTM